leaf-ui/js/onFunctions.js

```diff
@@ -1031,7 +1033,7 @@ paper.on('blank:pointerdown', function(evt, x, y) {
 });
 
 /**
- * 
+ * TODO: Unclear if this is actually doing anything - look into possible deletion
```

Comment:
I'd rather not leave TODOs in develop if we can avoid it. Maybe @amgrubb can chime in?